### ISSUE TITLE
homeport install: "Unable to find image bigeasy/env:latest" locally

### ISSUE THREAD

**Issue by vielmetti** [bigeasy/homeport#7]
I downloaded via git and put homeport in my source tree, and then tried to run it and got this:

```
Edwards-MacBook-Air:homeport emv$ ./homeport 
Unable to find image 'bigeasy/env:latest' locally
Pulling repository bigeasy/env
FATA[0002] Error: image bigeasy/env:latest not found    
error in run: exit status 1
```

I don't know if I missed a step prior to doing this.

**Comment by vielmetti** [comment]
Thanks for letting me be a guinea pig, @bigeasy .

**Comment by vielmetti** [comment]
I updated the build instructions with the new configure setup in pull request #15 and with that something is currently building.

**Comment by vielmetti** [comment]
```
Edwards-MacBook-Air:VirtualBox VMs emv$ docker run -it vielmetti/homeport-shell 
emv@1e094df3d654:/$ hostname
```

So some measure of success. Next step is to identify how to add a few more packages (mosh, screen).

**Comment by bigeasy** [comment]
Should be resolved by now, as we approach 1.0.
